packages/create-nx-workspace/src/create-workspace.ts

```diff
@@ -79,16 +75,14 @@ export async function createWorkspace<T extends CreateWorkspaceOptions>(
     }
   }
 
-  showNxWarning(name);
-
-  if (nxCloud && nxCloudInstallRes?.code === 0) {
-    printNxCloudSuccessMessage(nxCloudInstallRes.stdout);
-  }
-
   await recordStat({
     nxVersion,
     command: 'create-nx-workspace',
     useCloud: nxCloud,
     meta: messages.codeOfSelectedPromptMessage('nxCloudCreation'),
   });
```

Comment:
Move this out of here for now. Let's not collect stats on this.

I do think it is valuable to collect this information.. but we need to make sure other CLI authors do not do this by default.